spec/services/nytimes_service_spec.rb

```diff
@@ -9,22 +9,24 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("A Nuclear Deal Built on Coffee, Compromise and No Sleep")
 			)
 			expect(Article.first.url).to(
-				eq("http://www.nytimes.com/2015/04/02/us/california-imposes-first-" +
-					 "ever-water-restrictions-to-deal-with-drought.html")
+				eq("http://www.nytimes.com/2015/04/04/world/middleeast/" +
+           "an-iran-nuclear-deal-built-on-coffee-all-nighters-and-" +
+           "compromise.html")
 			)
 			expect(Article.first.abstract).to(
-				eq("The move by Gov. Jerry Brown comes as California’s drought has " +
-					 "reached near-crisis proportions after a winter that brought " +
-					 "record-low snowfalls.")
+				eq("In hashing out one of the hardest-to-negotiate arms" +
+           " control agreements in history, Iran and the United" +
+           " States took a simple approach: stay in the same hotel" +
+           " room until they figure out how to get along.")
 			)
 			expect(Article.first.desc_facet).to(
-				eq("")
+				eq("Nuclear Weapons,United States International Relations")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.